downstream/modules/hub/proc-create-api-token.adoc

```diff
@@ -13,14 +18,11 @@ In {HubName}, you can create an offline token using *Token management*. The offl
 . Click the btn:[Copy to clipboard] icon to copy the offline token.
 . Paste the token into a file and store in a secure location.
 
-[IMPORTANT]
+[NOTE]
 ====
-The offline token is a secret token used to protect your content. Store your token in a secure location.
+Your offline token expires after 30 days of inactivity. For more on obtaining a new offline token, see link:{URLHubManagingContent}/managing-cert-valid-content#con-offline-token-active_cloud-sync[Keeping your offline token active].
 ====
 
+.Next step
 The offline token is now available for configuring {HubName} as your default collections server or for uploading collections by using the `ansible-galaxy` command line tool.
```

Comment:
Similar to above, content seems more like .Results rather than .Next step as it is not an imperative statement requiring the customer to take an action. Of course, it's your final decision.